homeassistant/components/netatmo.py

```diff
@@ -50,7 +49,4 @@ def setup(hass, config):
         _LOGGER.error("Unable to connect to NatAtmo API")
         return False
 
-    for component in 'camera', 'sensor':
```

Comment:
It was creating issue as some person can have the weather station but does not have the camera, or they could have multiple camera and displaying only one.

My original goal was to use the discovery if no camera was settup but if one was setup, then do not use the discovery, but currently it did not work well as two feed of the same camera could be displayed